src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -208,7 +208,7 @@ def run(self, args, manage_args):
             code_dir = '/home/{cchq_user}/www/{deploy_env}/current'.format(
                 cchq_user=cchq_user, deploy_env=deploy_env)
         remote_command = (
-            '{code_dir}/python_env/bin/python {code_dir}/manage.py {args}'
+            'bash -c "cd {code_dir}; {code_dir}/python_env/bin/python {code_dir}/manage.py {args}"'
```

Comment:
Do we need to mention `code_dir` for python and manage.py if we have already used `cd`?

Reply:
good point - 7643f9381